src/Uno.UI/Microsoft/UI/Xaml/Controls/ScrollView/ScrollView.cs

```diff
@@ -0,0 +1,2692 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Numerics;
+using Microsoft.UI.Composition;
+using Microsoft.UI.Input;
+using Microsoft.UI.Private.Controls;
+using Microsoft.UI.Xaml.Controls.Primitives;
+using Microsoft.UI.Xaml.Media.Animation;
+using Microsoft.UI.Xaml.Input;
+using Uno;
+using Uno.Disposables;
+using Uno.UI.Helpers.WinUI;
+using Windows.Foundation;
+using Windows.System;
+using Windows.UI.ViewManagement;
+
+#if HAS_UNO_WINUI
+using PointerDeviceType = Microsoft.UI.Input.PointerDeviceType;
+#else
+using PointerDeviceType = Windows.Devices.Input.PointerDeviceType;
+#endif
+
+using static Microsoft/* UWP don't rename */.UI.Xaml.Controls._Tracing;
+
+namespace Microsoft.UI.Xaml.Controls;
+
+#if !__SKIA__
+[NotImplemented("__ANDROID__", "__IOS__", "__WASM__", "__MACOS__")]
+#endif
+public partial class ScrollView : Control, IScrollView
+{
+	// Change to 'true' to turn on debugging outputs in Output window
+	//bool ScrollViewTrace::s_IsDebugOutputEnabled{ false };
+	//bool ScrollViewTrace::s_IsVerboseDebugOutputEnabled{ false };
+
+	private const int s_noOpCorrelationId = -1;
+
+	public ScrollView()
```

Comment:
Didn't validated if already done, but this class has to [`RegisterAsScrollPort`](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.uielement.registerasscrollport?view=winrt-22621), update the [`ScrollOffsets `](https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/UIElement.cs#L76)  and then call [`InvalidateViewport`](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.frameworkelement.invalidateviewport?view=winrt-22621) in order to raise the `EffetiveViewportChange` event properly (and be usable with the `ItemRepeater`).